pulpcore/cli/common/context.py

```diff
@@ -797,18 +751,6 @@ class PulpContentContext(PulpEntityContext):
 EntityFieldDefinition = Union[None, str, PulpEntityContext]
 
 
-##############################################################################
-# Decorator to access certain contexts
-# DEPRECATED These are moved to generic.py and will be removed here.
-
-
-pass_pulp_context = click.make_pass_decorator(PulpContext)
-pass_entity_context = click.make_pass_decorator(PulpEntityContext)
-pass_repository_context = click.make_pass_decorator(PulpRepositoryContext)
-pass_repository_version_context = click.make_pass_decorator(PulpRepositoryVersionContext)
-pass_content_context = click.make_pass_decorator(PulpContentContext)
```

Comment:
Have we checked the other CLI plugins to see if they have switched over to the correct import spot?

Reply:
I switched them last week.